docs/contributing.md

```diff
@@ -89,23 +92,23 @@ Cargo will happily use up your entire hard drive. If this happens, run
 
 These are listed roughly in order of decreasing importance.
 
-1. Nearly any change to `jj`'s interface will require writing or updating
-golden tests that use the [`insta`](https://insta.rs/) crate.  To make
-this convenient, install the `cargo-insta` binary. Use `cargo insta tests`
-to run tests, and `cargo insta review` to update the golden tests.
+1. Nearly any change to `jj`'s CLI will require writing or updating snapshot
+   tests that use the [`insta`](https://insta.rs/) crate. To make this
+   convenient, install the `cargo-insta` binary. Use `cargo test`
+   to run tests, and `cargo insta review` to update the snapshot tests.
```

Comment:
`cargo insta test` is more appropriate here. `cargo test` would fail at the first assertion leaving remaining snapshots stale.

I tend to use `cargo insta test --workspace --test-runner nextest -- <testname>` while working on feature related to `<testname>`.

Reply:
+1 on the `insta test`. 

I think having `--test-runner nextest` in these instructions is unnecessary, then user will figure it out if they want to use it.

It seems to me that `cargo insta test -- <testname>` still compiles and run every test file. I *think* `cargo insta test --test <test_glob>` is better, but I'm still figuring it out.

Reply:
> `cargo insta test` is more appropriate here. `cargo test` would fail at the first assertion leaving remaining snapshots stale.

Oh, so that's what `cargo insta test` is for! I've seen it fail so often after running `cargo nextest` that I gave on it.

> I tend to use `cargo insta test --workspace --test-runner nextest -- <testname>` while working on feature related to `<testname>`.

Nice! I was running and old `cargo-insta` and had not thought about upgrading until I saw that it was missing most of those flags.

> I think having `--test-runner nextest` in these instructions is unnecessary, then user will figure it out if they want to use it.

If I had not been recommended `nextest` by someone (or some blog post or wherever I saw it), I'm not sure I would have gone looking for it, so I added a note about it.

> It seems to me that `cargo insta test -- <testname>` still compiles and run every test file. I _think_ `cargo insta test --test <test_glob>` is better, but I'm still figuring it out.

I introduced a compilation error a test in `lib/tests/` and `cargo insta test` didn't notice it.

Reply:
New version looks much better, thanks.